src/templates/components/emailContact/index.test.tsx

```diff
@@ -1,5 +1,6 @@
 import { render, screen } from '@testing-library/react'
 import { fireEvent, getByRole } from '@testing-library/dom'
+jest.mock('@/lib/utils/recordEvent')
```

Comment:
This and below is based on examples from https://github.com/aelbore/esbuild-jest/issues/26

I would appreciate a look to see if I am still in fact testing for the same things.

Reply:
![Screen Shot 2022-08-23 at 10 41 53 PM](https://user-images.githubusercontent.com/3916436/186339235-fae0ee6b-951d-4719-bf3b-60371d4d46e2.png)

Looks like the output matches the expected links-list-header:  `'t%24st.vet%3Dran%40va.gov',`
```
  event: 'nav-linkslist',
      'links-list-header': 't%24st.vet%3Dran%40va.gov',
      'links-list-section-header': 'Need more help?',
    })
```